examples/analytics.amp.html

```diff
@@ -99,52 +169,72 @@
       }
     }
   }
-</script>
-</amp-analytics>
-<!-- End AT Internet tracking -->
+</script> </amp-analytics>
+		<!-- End AT Internet tracking -->
 
-<!-- Burt tracking -->
-<amp-analytics type="burt">
-  <script type="application/json">
+		<!-- Burt tracking -->
+		<amp-analytics type="burt"> <script
+			type="application/json">
     {
       "vars": {
         "trackingKey": "B31VT4QH1BFR",
         "category": "sports",
         "subCategory": "golf"
       }
     }
-  </script>
-</amp-analytics>
-<!-- End Burt example -->
+  </script> </amp-analytics>
+		<!-- End Burt example -->
 
-<!-- Chartbeat tracking -->
-<amp-analytics type="chartbeat">
-<script type="application/json">
+		<!-- Chartbeat tracking -->
+		<amp-analytics type="chartbeat"> <script
+			type="application/json">
 {
   "vars": {
     "uid": "12345",
     "domain": "my-site.com",
     "sections": "section 1, section 2"
   }
 }
-</script>
-</amp-analytics>
-<!-- End Chartbeat example -->
+</script> </amp-analytics>
+		<!-- End Chartbeat example -->
 
-<!-- comScore UDM pageview tracking -->
-<amp-analytics type="comscore">
-<script type="application/json">
+		<!-- ColAnalytics pageview tracking -->
+		<amp-analytics type="colanalytics"> <script
+			type="application/json">
+{
+    "requests": {
+        "host": "https://${colurl}",
+        "base": "${host}/message",
+        "pageview": "${base}?cid=${id}&val_101=${canonicalPath}&val_101=${canonicalHost}&uuid=${uid}&val_101=${authors}&val_101=${zone}&val_101=${sponsorName}&val_101=${contentType}&val_101=${scrollTop}&val_101=${scrollHeight}&val_101=${decayTime}&val_101=${totalEngagedTime}&val_101=${documentReferrer}&val_101=${pageLoadTime}&val_108=${title}"
```

Comment:
I don't think you need these here. These are already defined in vendors.js. You just need to specify the vars. See googleanalytics tag for example.